packages/terra-switch/src/terra-dev-site/doc/switch/Switch.1.doc.mdx

```diff
@@ -0,0 +1,56 @@
+import { Badge } from 'terra-switch/package.json?dev-site-package';
+
+import DefaultSwitch from '../example/DefaultSwitch?dev-site-example';
+import SwitchDisabled from '../example/SwitchDisabled?dev-site-example';
+import SwitchLongTextLabel from '../example/SwitchLongTextLabel?dev-site-example';
+
+import SwitchPropsTable from 'terra-switch/src/Switch?dev-site-props-table';
+
```

Comment:
Looks like you forgot the `badge` here.

Reply:
Added in this [commit](https://github.com/cerner/terra-core/pull/2846/commits/1c002b1da07d995fc126d14539617437b6371dd7)